ibmdbpy/geoSeries.py

```diff
@@ -38,7 +38,23 @@ class IdaGeoSeries(ibmdbpy.IdaSeries):
     An IdaSeries whose column must have geometry type.
     It has geospatial methods based on Db2 Warehouse Spatial Extender (DB2GSE).
     
-    # TODO: add examples
+    Note on sample data used for the examples:
+    ------
+    Sample datasets available out of the box in Db2 Warehouse: GEO_TORNADO, GEO_COUNTY
+    Sample datasets which you can obtain yourself: SAMPLE_POLYGONS, SAMPLE_LINES,
+    SAMPLE_GEOMETRIES, SAMPLE_MLINES, SAMPLE_POINTS. See dedicated SQL script.
```

Comment:
Where can I find that dedicated SQL script?

Reply:
in the sampledata folder. It is written in the preliminary comment of the IdaGeoSeries class.

> See dedicated SQL script (ibmdbpy/sampledata/sql_script)

Reply:
I see. The problem is, if a user is reading the documentation at https://pythonhosted.org/ibmdbpy/geoSeries.html, they're far away from the repository. In fact, if they have installed ibmdbpy, they might not even have the script anywhere on their hard drive! There should at least be a link to it.

If you want to test the documentation, `pip install sphinx` and then run `make` in the `docs` folder.

Reply:
Do you mean, impossible to build the documentation at all, or impossible to do it from my computer using sphinx?

Reply:
With the code on the master (and your) branch, not at all, unfortunately.